tests/rest/admin/test_event_reports.py

```diff
@@ -290,6 +290,60 @@ def test_from_is_negative(self):
         self.assertEqual(400, int(channel.result["code"]), msg=channel.result["body"])
         self.assertEqual(Codes.INVALID_PARAM, channel.json_body["errcode"])
 
+    def test_next_token(self):
+        """
+        Testing that ``next_token`` appears at the right place
```

Comment:
```suggestion
        Testing that `next_token` appears at the right place
```

We generally use markdown form in docstrings and comments.